notebooks/json_blocks/BLOCK-T539.ipynb

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            "description": "Day of the year for the reference state.",
</pre><p>Isn't this day_obs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>